test/relay.test.js

```diff
@@ -136,10 +137,24 @@ describe('relay', function () {
       }
     });
 
+    viewerType = new GraphQLObjectType({
+      name: 'Viewer',
+      description: 'root viewer for queries',
+      fields: () => ({
+        id: globalIdField('Viewer'),
+        name: {
+          type: GraphQLString,
+          resolve: () => 'Viewer!'
+        }
+      }),
+      interfaces: [nodeInterface]
+});
+
     nodeTypeMapper.mapTypes({
       [User.name]: userType,
       [Project.name]: projectType,
-      [Task.name]: taskType
+      [Task.name]: taskType,
+      viewer: viewerType
```

Comment:
Yeah, it's already normalizing the types here https://github.com/mickhansen/graphql-sequelize/blob/master/src/relay.js#L25 but I think its safe for right now. Not a huge issue changing that.